rust/Cargo.toml

```diff
@@ -10,7 +10,7 @@ license = "Apache-2.0"
 name = "minify-js"
 readme = "README.md"
 repository = "https://github.com/wilsonzlin/minify-js"
-version = "0.2.4"
```

Comment:
We use a versioning script that prepares and changes files for a version bump across all projects in a standalone commit, so this should be unchanged.